modules/aruco/test/test_charucodetection.cpp

```diff
@@ -678,7 +611,7 @@ TEST(Charuco, testCharucoCornersCollinear_false)
 }
 
 // test that ChArUco board detection is subpixel accurate
-TEST(Charuco, testBoardSubpixelCoords)
+TEST(Charuco, DISABLED_testBoardSubpixelCoords)  // FIXIT: https://github.com/opencv/opencv_contrib/pull/3213
```

Comment:
@AleksandrPanov Disabled this test. It is completely rewritten in 4.x branch (see #3213). Main question is "why?"